MAINTAINERS

```diff
@@ -138,10 +138,10 @@ W: http://plugins.geany.org/geanyprj.html
 S: Odd Fixes
 
 geanypy
-P: Lex Trotman <elextr@gmail.com>
-M: Lex Trotman <elextr@gmail.com>
```

Comment:
On 10 June 2016 at 21:36, Jiří Techet <notifications@github.com> wrote:

> In MAINTAINERS
> <https://github.com/geany/geany-plugins/pull/435#discussion_r66599694>:
>
> > @@ -138,10 +138,10 @@ W: http://plugins.geany.org/geanyprj.html
> >  S: Odd Fixes
> >
> >  geanypy
> > -P: Lex Trotman <elextr@gmail.com>
> > -M: Lex Trotman <elextr@gmail.com>
>
> Let's get technical and review the patch :-).
>
> I actually agree with the "-" part of the patch - having GP version of the
> plugin maintained by someone else than the upstream author is less than
> ideal. IMO it would be best having Matthew Brush at the "+" side of the
> patch if @codebrainz <https://github.com/codebrainz> is willing to
> maintain it.
>
> Otherwise I think it doesn't make sense to keep an unmaintained copy of a
> maintained plugin in GP.
>
​In that case you better remove quite a lot of other plugins that are also
unmaintained.

There is nothing to review.  If a later patch adds a maintainer, fine.


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/geany/geany-plugins/pull/435/files/e0d77167b2d9550fbd2e689347a80fb97371004b#r66599694>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAxgTZWDxN23YrDKLz78C9qDEJCfRBG8ks5qKUwygaJpZM4Ix325>
> .
>